### PR TITLE
Cura 6013 rate packages

### PR DESCRIPTION
**Pull request by nallath**
CURA-6013

### CODE REVIEW

File: cura/API/Account.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def __init__(self, application: "CuraApplication", parent = None) -> None:
             OAUTH_SERVER_URL= self._oauth_root,
             CALLBACK_PORT=self._callback_port,
             CALLBACK_URL="http://localhost:{}/callback".format(self._callback_port),
-            CLIENT_ID="um---------------ultimaker_cura_drive_plugin",
+            CLIENT_ID="um----------------------------ultimaker_cura",
```
Comment (LipuFei) [inline cura/API/Account.py]: @ChrisTerBeke Do you know what's this `CLIENT_ID` for? Why there's so many dashes in it?

Comment (ChrisTerBeke) [reply cura/API/Account.py]: Our OAuth2 server specifies an application client identifier of 44 characters should be used. This is the exact ID that is registered in the authorization server for Cura.

File: plugins/Toolbox/resources/qml/RatingWidget.qml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                    background: UM.RecolorImage
+                    {
+                        source: UM.Theme.getIcon(control.isStarFilled ? "star_filled" : "star_empty")
+
+                        // Unfilled stars should always have the default color. Only filled stars should change on hover
+                        color:
+                        {
+                            if(!ratingWidget.canRate)
+                            {
+                                return "#5a5a5a"
```
Comment (alekseisasin) [inline plugins/Toolbox/resources/qml/RatingWidget.qml]: get color from Theme, the dark them has another background

File: plugins/Toolbox/resources/qml/SmallRatingWidget.qml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        id: starIcon
+        source: UM.Theme.getIcon("star_filled")
+        color: model.user_rating == 0  ? "#5a5a5a" : UM.Theme.getColor("primary")
+        height: UM.Theme.getSize("rating_star").height
+        width: UM.Theme.getSize("rating_star").width
+    }
+
+    Label
+    {
```
Comment (alekseisasin) [inline plugins/Toolbox/resources/qml/SmallRatingWidget.qml]: + renderType: Text.NativeRendering

**Pull request merged**